application/templates/static_site/export/measure_export.html

```diff
@@ -7,398 +7,361 @@
-                        <div id="publisher-value">{{ measure_page.primary_data_source.publisher.name }}</div>
+                    <div id="secondary-source-publication-release" class="heading-small">Publication release date</div>
+                    {% if measure_page.secondary_data_source.publication_date %}
+                        <div id="secondary-source-publication-release-value">{{ measure_page.secondary_data_source.publication_date }}</div>
                     {% else %}
                         <div>[...]</div>
                     {% endif %}
                 </div>
 
                 <div style="padding-bottom: 10px">
-                    <div id="source-url" class="heading-small">Link to data source</div>
-                    {% if measure_page.primary_data_source and measure_page.primary_data_source.source_url %}
-                        <div id="source-url-value" >{{ measure_page.primary_data_source.source_url }}</div>
+                    <div id="secondary-source-publication-frequency" class="heading-small">Publication frequency</div>
+                    {% if measure_page.secondary_data_source.frequency_of_release.description %}
+                        <div id="secondary-source-publication-frequency-value">{{ measure_page.secondary_data_source.frequency_of_release.description }}</div>
                     {% else %}
                         <div>[...]</div>
                     {% endif %}
                 </div>
 
                 <div style="padding-bottom: 10px">
-                    <div id="publication-release-date" class="heading-small">Source data publication date</div>
-                    {% if measure_page.primary_data_source and measure_page.primary_data_source.publication_date %}
-                        <div id="publication-release-date-value">{{ measure_page.primary_data_source.publication_date }}</div>
+                    <div id="secondary-source-type-of-data" class="heading-small">Type of data (Admin or Survey)</div>
+                    {% if measure_page.secondary_data_source.type_of_data %}
+                        <div id="secondary-source-type-of-data-value" {{ measure_page.secondary_data_source.type_of_data| join_enum_display_names(' and ') }} data></div>
                     {% else %}
                         <div>[...]</div>
                     {% endif %}
                 </div>
 
                 <div style="padding-bottom: 10px">
-                    <div id="notes-on-corrections-or-update" class="heading-small">Corrections or updates</div>
-                    {% if measure_page.primary_data_source and measure_page.primary_data_source.note_on_corrections_or_updates %}
-                        <div id="notes-on-corrections-or-update-value" >{{ measure_page.primary_data_source.note_on_corrections_or_updates | render_markdown }}</div>
+                    <div id="secondary-source-type-of-satistic" class="heading-small">Type of statistic</div>
```

Comment:
typo: `secondary-source-type-of-satistic` should end with `statistic`

Reply:
Hehe. This has been there for ages (I just moved the lines). Not sure these IDs are even needed.